server/events/post_workflow_hooks_command_runner.go

```diff
@@ -106,6 +106,16 @@ func (w *DefaultPostWorkflowHooksCommandRunner) runHooks(
 		if hookDescription == "" {
 			hookDescription = fmt.Sprintf("Post workflow hook #%d", i)
 		}
+		shell := hook.Shell
+		if shell == "" {
+			ctx.Log.Debug("Setting shell to default")
```

Comment:
Slight nitpick, can we include the default in the logs for the user:
```
if shell == "" {
    shell = "sh"
    ctx.Log.Debugf("Setting shell to default: %q", shell)
}
```

Reply:
Done